worlds/peaks_of_yore/__init__.py

```diff
@@ -0,0 +1,172 @@
+import random
+from typing import Any
+
+from worlds.AutoWorld import World, WebWorld
+from BaseClasses import Tutorial, Item, Region, ItemClassification
+from .options import PeaksOfYoreOptions, Goal
+from .data import full_item_list, full_location_list
+from .items import PeaksOfYoreItem
+from .locations import get_locations, get_location_names_by_type, PeaksOfYoreLocation
+
+
+class PeaksOfWeb(WebWorld):
+    theme = "stone"
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Peaks of Yore Mod.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["c0der23"]
+    )]
+
+
+class PeaksOfWorld(World):
+    """
+    Peaks of Yore is a first-person physics-based "climb-em-up" adventure set in 1887.
+    Steel your nerves and perfect your climbing skills as you ascend the rock wall, traverse difficult routes, and encounter many challenges and obstacles.
+    """
+    game = "Peaks of Yore"
+    options_dataclass = PeaksOfYoreOptions
+    options: PeaksOfYoreOptions
+    web = PeaksOfWeb()
+    item_name_to_id = {item.name: item.id for item in full_item_list}
+    location_name_to_id = {location.name: location.id for location in full_location_list}
+    topology_present = True
+    location_count = 0
+    peaks_in_pool = []
+    artefacts_in_pool = []
+
+    def create_item(self, name: str) -> Item:
+        classification: ItemClassification = ItemClassification.filler
+        item_entry = [item for item in full_item_list if item.name == name][0]
+        if item_entry.type in ["Book", "Tool"]:
+            classification = ItemClassification.progression
+        return PeaksOfYoreItem(name, classification, self.item_name_to_id[name], self.player)
+
+    def create_extra_item(self):
+        choices = ["Extra Rope", "Extra Coffee", "Extra Chalk", "Extra Seed"]
+        return self.create_item(random.choice(choices))
```

Comment:
```suggestion
        return self.create_item(self.random.choice(choices))
```
Please use the world's random object, it'll make it more deterministic